### ISSUE TITLE
Train on LIDC and test on LUNA

### ISSUE THREAD

**Issue by vankhoa21991** [MIC-DKFZ/nnDetection#12]
Hello, as I read your paper, I understand that you trained your model on a pool of dataset including LIDC, then you test on LUNA16. I have two questions for this:

1/ The figure 2 in the paper means you use all 888 images of LUNA as the test set? If not how did you have this result?
2/ LUNA16 is a subset of LIDC, even though you explained on the paper but I still doubt that two datasets are mostly overlapped, could you explain a bit more on this?

Thanks for your help.

**Comment by mibaumgartner** [comment]
Hi @vankhoa21991 ,

thanks for your question. In our setting, there is a difference between a `test set` and the `test pool` since the development of nnDetection contains a "meta optimization" process. During this process, we developed a set of rules which deduce a set of rules to adapt to unseen data sets / data sets with different Data Fingerprint.

Data sets that are part of the validation pool have a separated `test set`, i.e. we keep a certain amount of the data completely separated from the development process of nnDetection and only evaluated this subset for our final reported results. This type of evaluation is commonly found and measures the generalization of the trained model to unseen data.

The `test pool` contains data sets that were not part of the development process of nnDetection but have a different Data Fingerprint. These data sets are meant to measure the generalization of our developed rules to new data sets.
As we explain in our paper, while the LUNA16 data is a subset of LIDC it has different scan selection criteria and different annotations. These result in different Data Fingerprints and also different training/inference configurations. Please note that the models trained on LIDC are *not* evaluated on LUNA16, all models which were evaluated on LUNA16 were only trained on LUA16 data (see exact evaluation protocol below).

LUNA16 protocol: According to the official guidelines, LUNA16 should be trained in a 10 Fold cross-validation fashion, i.e. there are 9 subsets to train and 1 subset to predict. The subset to predict changes with every fold, so after training 10 times, the whole data set was predicted at some point.

Best,
Michael

**Comment by Thibescobar** [comment]
Pour poster sur un forum :

Hello, 

Thank you for this explanation. To be sure I understand well. 

The pre-training rule-based estimation of hyperparameters have been done using the full training dataset including LIDC, so we consider the fingerprints not to be estimated from the LUNA16 data (only) at this stage. So no significant data leakage before training ? 

As for evaluation, the cross validation splits have been used so we have out-of-training predictions for all LUNA16 data ? 

For this latter step, did the sweep step (post-training empirical optimization of inference hyperparameters) not introduce some form of significant data leakage ? 

Thank you very much for your help.
